### DISCUSSION TITLE
When will the new version be released? Sniper mode is not supported in v2.5.3

### DISCUSSION THREAD

**Discussion by dean2021** [projectdiscovery/nuclei#1211] [Q&A] [answered]
When will the new version be released? Sniper mode is not supported in v2.5.3

**Comment by ehsandeep** [comment, marked as answer]
@dean2021 `sniper` is replaced with `batteringram` as the default payload type and docs are updated accordingly - https://github.com/projectdiscovery/nuclei-docs/commit/cb22a26285c57dacf2a8f5a42b23cbf586e41cf6

More context - https://github.com/projectdiscovery/nuclei/issues/814
